framework/src/systems/NonlinearSystemBase.C

```diff
@@ -2816,11 +2816,20 @@ NonlinearSystemBase::computeJacobianInternal(const std::set<TagID> & tags)
       // they are almost always used in conjunction with Kernels
       ComputeJacobianForScalingThread cj(_fe_problem, tags);
       Threads::parallel_reduce(elem_range, cj);
+      // BWS TODO Add a call to FEProblemBase::checkExceptionAndStopSolve() here
```

Comment:
@lindsayad I need to delete this before we merge this PR. You took care of that a few lines below. I want to see if the currently running tests pass before I go update this.

Reply:
K I removed it